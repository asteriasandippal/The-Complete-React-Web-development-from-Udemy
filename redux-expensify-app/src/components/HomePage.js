import React from 'react';
import ExpenseList from './ExpenseList';

const HomePage = () => (
    <div>
        <ExpenseList/>
    </div>
);

export default HomePage;
