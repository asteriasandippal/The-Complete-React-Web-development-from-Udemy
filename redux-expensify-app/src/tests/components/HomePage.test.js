import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import HomePage from '../../components/HomePage';

test('should render HomePage correctly', () => {
    const wrapper = shallow(<HomePage />);
    expect(toJson(wrapper)).toMatchSnapshot();
});
