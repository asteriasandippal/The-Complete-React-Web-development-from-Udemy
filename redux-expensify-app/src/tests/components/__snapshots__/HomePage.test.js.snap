// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render HomePage correctly 1`] = `
<div>
  <Connect(ExpensesSummary) />
  <Connect(ExpenseListFilter) />
  <Connect(ExpenseList) />
</div>
`;
